molecule/default/molecule.yml

```diff
@@ -19,5 +19,9 @@ platforms:
       - /tmp
 provisioner:
   name: ansible
```

Comment:
All of our other roles only have ansible here, nothing else.  Are you sure the rest of this is needed?